packages/client/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "@lens-protocol/tsconfig/base.json",
   "compilerOptions": {
     "module": "ESNext",
+    "lib": ["DOM", "ESNext"],
```

Comment:
nit: personally would just copy the `fetch` type definitions into some `typings` folder with a link to the issue as `DOM` opens the possibility to use some DOM-specific api and break node.js integrations down the line.